Makefile

```diff
@@ -30,15 +30,18 @@ build: ## Builds the all docker setup.
 	docker compose build
 
 run-devserver: ## Runs the dev-server
-	docker compose up angular-build -d
+# Add container label that it hasn't completed first run
+	COMPLETED_FIRST_RUN=0 docker compose up angular-build -d
 	$(MAKE) update.package
 	docker cp oppia-angular-build:/app/oppia/node_modules .
 	docker compose stop angular-build
-	docker compose up dev-server -d --no-deps
+	COMPLETED_FIRST_RUN=0 docker compose up dev-server -d --no-deps
 	$(MAKE) update.requirements
```

Comment:
there are some other issues as well.
1. update.requirements don't require internet as it's simple using cache. 
2. start-devserver (used in `run-offline`) requires internet if firebase-emulator and some other conatiners are not found! - It can be fixed by checking if container exisits.